backend/controller/sql/conn.go

```diff
@@ -19,13 +27,23 @@ type ConnI interface {
 	Begin(ctx context.Context) (pgx.Tx, error)
 }
 
+type cronjobsQueries = cronjobssql.Queries
+type leasesQueries = leasessql.Queries
+
 type DB struct {
 	conn ConnI
 	*Queries
+	*cronjobsQueries
+	*leasesQueries
```

Comment:
I don't understand why these are still in here? We should be splitting the queries into their own packages as well.

Reply:
Oh you're going to move these in a followup?

Reply:
I had to incorporate all of these in order to make sure the `Tx` behavior remains atomic. Otherwise, we'd have to split all the queries across several transactions, which would defeat the purpose of txns :/

Reply:
Yes, but it would be preferable to figure out a way to restructure the packages to support this. Having the different DALs still be tightly coupled defeats the purpose of this refactoring.

Reply:
That makes sense. I'll look into that now

Reply:
I changed it to effectively follow https://docs.sqlc.dev/en/stable/howto/transactions.html. I'm not totally confident that the current error handling (panics if the rollback fails) is what we actually want... what do you think?